README.rst

```diff
@@ -133,6 +133,62 @@ input queries before sending them to the API:
 Run ``python -m autoextract --help`` to get description of all supported
 options.
 
+Errors
+~~~~~~
+
+The following errors could happen while making requests:
+
+- Network errors
+- `Request-level errors`_
+    - Authentication failure
+    - Malformed request
+    - Too many queries in request
+    - Request payload size is too large
+- `Query-level errors`_
+    - Downloader errors
+    - Proxy errors
+    - ...
+
+Some errors could be retried while others don't.
+
+For example,
+you could retry a query with a Proxy Timeout error
```

Comment:
If changing the above, then this one could be changed as well, I think..

Reply:
Done.